kyverno-policy-deployment/README.md

```diff
@@ -0,0 +1,71 @@
+# Handling Kyverno Policy Deployment
+
+Our Kyverno policies have the following properties:
+1. They are owned by different teams
+2. The need to be deployed to different environments & on different providers
+3. They required different testing setups according to 1. and 2.
+
+Up until now we kept these policies in the same repository called `kyverno-policies`.
+Now that we will have to rename the repository to upgrade Kyverno is an opportune time to discuss the future structure of our Kyverno policies.
+
+## Proposals
+
+These proposals are roughly ordered by the amount of repositories they would introduce by increasing order.
+
+## Keep all policies in the same repository
+It's simple, everything stays the same and `kyverno-policies` is just renamed.
+### Pros
+- Not a lot of effort
+- Not much communication to make a change
+- Single source of truth for all policies
+### Cons
+- Testing setup is complex as all different environments need to be covered in one repo
+- Ownership is difficult as multiple teams own parts of the repo
```

Comment:
@MarcelMue during a discussion with @stone-z we agreed that if you mix all kind of use-cases in one repo there is a risk that you will build complex interdependencies of policies. Stacking policies on top of each other and building a hierarchy will be a bad design. Policies split by use-case will be leaner and easier to test with less side-effects. Still @stone-z sees a lot of risks in side-effects if policies are split and not tested as a whole. Imo it would be good to define design principals and best practices that make sure policies are lean (not too complex), narrow in scope and independent.

Reply:
Let's discuss tomorrow in the session how we want to progress forward. I want to initially _exclude_ solution proposals and then talk about how we can make the remaining ones work.

Coming up with design principles makes sense to me but it's not something we can do immediately right now IMO. So talking about when & how we come up with them makes sense to me.